cmd/deploy.go

```diff
@@ -126,7 +126,18 @@ func runDeploy(commandContext *cmdctx.CmdContext) error {
 
 	var image *docker.Image
 
-	if imageRef, _ := commandContext.Config.GetString("image"); imageRef != "" {
+	configimageRef, _ := commandContext.Config.GetString("image")
+	appimageRef := commandContext.AppConfig.Build.Image
```

Comment:
`Build` might be a null pointer here if there's no config file or if there's no build section

Reply:
On it.